### PR TITLE
PHEx Infinity

### PR DESCRIPTION
**Pull request by afkvido**
- [ ] #104

### CODE REVIEW

**Comment by afkvido** [comment]
Hell yea for [c888ea9](https://github.com/ProdigyPNP/ProdigyMathGameHacking/pull/105/commits/c888ea9e3ddeaa8b67c578d0b18897c53cf58aeb)

**Comment by Erisfiregamer1** [comment]
Dang, that was fast.
Some things to check:
- Is MV3 going to break this worse?
- Does it even WORK??
- Are we ready for this?

**Comment by Erisfiregamer1** [comment]
Testing extension, if it *actually* works I'm going to pull out of draft.

**Comment by afkvido** [comment]
We **must** remove the Build becasue it **does NOT** sign the CRX.

**Review by Erisfiregamer1 — APPROVED** [review]
Once you iron out the bug with it only working once after install, ready for merge.

**Comment by afkvido** [comment]
> Once you iron out the bug with it only working once after install, ready for merge.

_i think you're requesting changes_

**Comment by Erisfiregamer1** [comment]
nah I'm saying "it's good to go just fix this ONE THING"

**Comment by afkvido** [comment]
that moment when the one error fucks up the whole thing
wait a sec thats called programming

**Comment by afkvido** [comment]
> Dang, that was fast. Some things to check:
> 
> * Is MV3 going to break this worse?
> * [ ]  Does it even WORK??
> * [x]  Are we ready for this?
> 
> Also promoting to major because HOLY FUCK it's revamping how the gamecode is loaded.

# Hell yea

**Comment by afkvido** [comment]
@Erisfiregamer1 @donaldli2020 

**Critical-High Priority**.
I've already updated the P-NP instances to match 2.2.0 in preparation for this, let's go, come on, let's go, yes, I am rushing yall.

Ok whatever imma just uhh do it myself

File: PHEx/src/disableIntegrity.js
Comment (donaldli2020) [reply PHEx/src/disableIntegrity.js:26]: stfu

**Comment by afkvido** [comment]
https://github.com/ProdigyPNP/InfiniteZero/commit/79987d495a2e84a2bde0d35210267cb8dc888d15

This will be helpful.

**Comment by afkvido** [comment]
@Erisfiregamer1 @donaldli2020

How does this look?
I think that this **does** work from my testing.
Please also test it, because I ~~have paranoia and PTSD from pushing bad updates~~ would like to confirm that this works for _everyone_.

**Comment by afkvido** [comment]
Also, please **do not merge** this _yet_. 

Once y'all have reviewed the PR, I need to sign the CRX.

**Comment by afkvido** [comment]
Wooo, Project Infinite Zero is almost done!

**Comment by afkvido** [comment]
HOL UP gotta fix a thing on infinite zero

**Review by Erisfiregamer1 — APPROVED** [review]
It passed the 3-reload test.
I'm pushing the big green button. HERE GOES FUCKING NOTHING.

**Comment by afkvido** [comment]
> We've got a CORS error, but that won't hurt. Here's the refrence: `Access to fetch at 'https://pnp5.prodigypnp.repl.co/game.min.js' from origin 'https://math.prodigygame.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.` (fixed on reload)

If it works on reload, then we should be all good.

**Comment by afkvido** [comment]
> It passed the 3-reload test. I'm pushing the big green button. HERE GOES FUCKING NOTHING.

**HELL YEA DAWG**

**Pull request merged**
